modules/core/src/main/java/org/apache/ignite/internal/processors/security/IgniteSecurityProcessor.java

```diff
@@ -188,6 +193,20 @@ public IgniteSecurityProcessor(GridKernalContext ctx, GridSecurityProcessor secP
         }
     }
 
+    /**
+     * Updates the package access property to specify the internal Ignite package.
+     */
+    private void updatePackageAccessProperty() {
+        String packAccess = Security.getProperty("package.access");
+
+        if (packAccess != null && !packAccess.isEmpty()) {
+            if (!packAccess.contains(IGNITE_INTERNAL_PACKAGE))
+                Security.setProperty("package.access", packAccess + ',' + IGNITE_INTERNAL_PACKAGE);
```

Comment:
What if some code enables sandbox, starts Ignite, stops it, then disables sandbox and starts Ignite again in the same JVM? I think we should clear `package.access` on processor stop (perhaps count started with sandbox nodes, and clear property after counter equals to 0 under some synchronization on static field)